cmd/arrai/version.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+var versionCommand = &cli.Command{
+	Name:    "version",
+	Aliases: []string{"v"},
+	Usage:   "display arrai version information",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "verbose",
+			Aliases: []string{"v"},
+			Usage:   "display full arrai version information",
+		}},
+	Action: version,
+}
+
+func version(c *cli.Context) error {
+	if c.Bool("verbose") {
+		fmt.Printf("Version    : %s\n", Version)
+		fmt.Printf("Git commit : %s\n", GitFullCommit)
+		fmt.Printf("Date       : %s\n", BuildDate)
+		fmt.Printf("OS/arch    : %s\n", BuildOS)
+		fmt.Printf("Go version : %s\n", GoVersion)
+	} else {
+		fmt.Printf("Version    : %s\n", Version)
+		fmt.Printf("OS/arch    : %s\n", BuildOS)
```

Comment:
I prefer this single-line format for the simple version:
```shell
arrai 0.123.0 linux/amd64
```

There's plenty of precedent for this:
```
$ go version
go version go1.14.3 linux/amd64
$ git version
git version 2.24.1
$ node --version
v10.15.2
$ python --version
Python 2.7.17
$ ruby --version
ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-linux-gnu]
$ erl -version
Erlang (SMP,ASYNC_THREADS) (BEAM) emulator version 10.4.4
$ mono-csc --version
Mono C# compiler version 5.18.0.240
$ sqlite3 --version
3.29.0 2019-07-10 17:32:03 fc82b73eaac8b36950e527f12c4b5dc1e147e6f4ad2217ae43ad82882a88alt1
$ rstudio --version
1.3.959
```

There are numerous counterexamples, but they seem to fall mostly into the category of copyright notices on GNU FSF applications, e.g.:

```
$ c++ --version
c++ (Ubuntu 9.2.1-9ubuntu2) 9.2.1 20191008
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

...a precedent I do _not_ want to follow.

Reply:
Current output for simple version:
```
arrai version 0.30.0 darwin/amd64
```